### ISSUE TITLE
Spain translate??

### ISSUE THREAD

**Issue by rafacruz-te20** [venturedrake/laravel-crm#50]
- Laravel CRM Version: Last version
- PHP Version: 8.1.23

### Description:
Is possible to translate the crm to spanish?

**Comment by andrewdrake** [comment]
Yes of course, you can create a lang folder called "es", copy the files in the lang/en folder and replace each array value with the Spanish version. If you like please submit a pull request and I can merge this into the project also. This is something I do plan on working on in the future also, adding of various language files

Eg:

`return [
    'created_last_14_days' => 'creado los últimos 14 días,
    'dashboard' => 'panel',
    'deal' => 'trato',
   ....
];`

It would look like this:

![image](https://github.com/venturedrake/laravel-crm/assets/2742557/6412d2e3-be06-483a-afa3-cf50558f98a7)

**Comment by jeovajireh** [comment]
> Yes of course, you can create a lang folder called "es", copy the files in the lang/en folder and replace each array value with the Spanish version. If you like please submit a pull request and I can merge this into the project also. This is something I do plan on working on in the future also, adding of various language files
> 
> Eg:
> 
> `return [ 'created_last_14_days' => 'creado los últimos 14 días, 'dashboard' => 'panel', 'deal' => 'trato', .... ];`
> 
> It would look like this:
> 
> ![image](https://private-user-images.githubusercontent.com/2742557/304577944-6412d2e3-be06-483a-afa3-cf50558f98a7.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA1NzYyMjgsIm5iZiI6MTczMDU3NTkyOCwicGF0aCI6Ii8yNzQyNTU3LzMwNDU3Nzk0NC02NDEyZDJlMy1iZTA2LTQ4M2EtYWZhMy1jZjUwNTU4Zjk4YTcucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTEwMiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDExMDJUMTkzMjA4WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ODMwMjYzMTEyZGJmOThjYzNjYWQyNDgxNjNkZjQxZWIxODEwYjM0YTMyYzkyMTc0MWVjYTc4MWM2YWU2M2RkYSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.3tGeBMBK7-H0SLsaXgBrDfIjES7ZiASCyKG1-IPCN-g)

Hello! I'm doing this for Portuguese and I'm not having success, it doesn't appear in settings, only English, and if I translate the file in the "en" folder it doesn't reflect in the interface either.

Thanks
